src/site/layout/mod.rs

```diff
@@ -8,7 +8,7 @@ use crate::config::{analytics, theme, Config};
 use crate::errors::*;
 use axohtml::{html, text, unsafe_text};
 
-pub fn build(config: &Config, content: String, is_index: bool) -> Result<String> {
```

Comment:
are we not using the index to build layout anymore?

Reply:
nop, we need the JS in the artifacts page too

Reply:
Fixed